dir.props

```diff
@@ -499,13 +506,17 @@
 
   <!-- Default Test platform to deploy the netstandard compiled tests to -->
   <PropertyGroup>
-    <DefaultTestTFM>netcoreapp1.1</DefaultTestTFM>
+    <DefaultTestTFM>netcoreapp2.0</DefaultTestTFM>
     <TestTFM Condition="'$(TestTFM)'==''">$(DefaultTestTFM)</TestTFM>
     <!-- we default FilterToTestTFM to DefaultTestTFM if it is not explicity defined -->
     <FilterToTestTFM Condition="'$(FilterToTestTFM)'==''">$(DefaultTestTFM)</FilterToTestTFM>
   </PropertyGroup>
 
   <PropertyGroup>
+    <TestNugetTargetMoniker Condition="'$(TestTFM)' == 'netcoreapp2.0'">.NETCoreApp,Version=v2.0</TestNugetTargetMoniker>
```

Comment:
Consider adding a comment here that says this belongs in https://github.com/dotnet/buildtools/blob/66dc7235af5e0a9f3edcaa929da7f0a5fdd21259/src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets#L69